### PR TITLE
Convert "objectBoundingBox" to "userSpaceOnUse" before transforming gradients

### LINKED ISSUE #226: Handle objectBoundingBox gradients
#223 improves processing somewhat but does not result in correct handling of gradients with objectBoundingBox. Uncomment the objectBoundingBox tests in `svg_test.py` and fix. 

Ref https://github.com/googlefonts/nanoemoji/blob/6675c6fd0d11dc7aece5fda779ef25c1b7291071/src/nanoemoji/color_glyph.py#L91-L112

### PR DESCRIPTION
**Pull request by anthrotype**
Fixes #226

(note the base branch is `ungroup`, not `main`)

### CODE REVIEW

File: src/picosvg/svg.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
     xlinkns,
     parse_css_declarations,
     parse_view_box,
+    _LinkedDefault,
```
Comment (rsheeter) [inline src/picosvg/svg.py:42]: might as well make it linked_default?

**Comment by anthrotype** [comment]
here's how the test files `transform-{linear,radial}-objectbbox-nano.svg` render after this patch

image


image

File: src/picosvg/svg.py (hunk 1 above)
Comment (anthrotype) [reply src/picosvg/svg.py:42]: it's a class, usually in python you use TitleCase for those

Comment (rsheeter) [reply src/picosvg/svg.py:42]: Oh it's a class, then just remove the _?

**Review by rsheeter — APPROVED** [review]

File: src/picosvg/svg.py (hunk 1 above)
Comment (anthrotype) [reply src/picosvg/svg.py:42]: ok.. it's just that it's kind of a hack so i'd rather mark it as private (`_` prefixed)

Comment (rsheeter) [reply src/picosvg/svg.py:42]: Your very first use is importing it somewhere else so ... it's not *really* private

**Pull request merged**